lib/msal-common/src/utils/Constants.ts

```diff
@@ -37,9 +38,20 @@ export const Constants = {
     AUTHORIZATION_PENDING: "authorization_pending",
     NOT_DEFINED: "not_defined",
     EMPTY_STRING: "",
-    FORWARD_SLASH: "/" 
+    FORWARD_SLASH: "/"
 };
 
+export const OIDC_DEFAULT_SCOPES = [
+    Constants.OPENID_SCOPE,
+    Constants.PROFILE_SCOPE,
+    Constants.OFFLINE_ACCESS_SCOPE
```

Comment:
It's a breaking change to behavior, but adding email here would be _really_ nice. I've never understood why MSAL doesn't request the user's email address.

Reply:
@sameerag @jasonnutter possibly something to add to next major version?

Reply:
Why would this be breaking?

Reply:
Seems additive/incremental.

Reply:
Adding the email claim out of no where will slightly change the consent screen, and would trigger new consent prompts for existing apps.

Reply:
Ah, makes sense.

Reply:
> @sameerag @jasonnutter possibly something to add to next major version?

Yeah, worth noting.